frontent/src/Website/CommentBox.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import './MovieDetails.css';
+import logoimage from "../aessest/UserPNG.png";
+
+const CommentBox = () => {
+  const [commentValue, setCommentValue] = useState('');
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    const fetchComments = async () => {
+      try {
+        const url = "http://localhost:8000/comments";
+        const response = await fetch(url);
+        const data = await response.json();
+        setComments(data); 
+        console.log(data); 
+      } catch (error) {
+        console.error("Error fetching comments", error);
+      }
+    };
+    fetchComments();
+  },[]); 
+
+  const handleCommentChange = (e) => {
+    setCommentValue(e.target.value);
+  };
+
+  const handleCommentSubmit = async () => {
+    if (!commentValue.trim()) {
+      alert("Please enter a valid comment");
+      return;
+    }
+    try {
+      const url = "http://localhost:8000/comments";
```

Comment:
the url for fetching and posting comments is the same cant we extract the url into a variable and use that variable everywhere?